java/daikon/dcomp-dummy/README

```diff
@@ -0,0 +1,36 @@
+This directory contains the initial, dummy version of java.lang.DCRuntime.
+
+Post Java 8 there are increased security checks when loading JDK classes. In particular, the
```

Comment:
This text appears redundantly in 3 places.  Please retain the text in just one place, and have the others point to the canonical explanation.

Reply:
Consolidated the information in dcomp/README.  The two READMEs in dcomp-dummy and dcomp-transfer now just point to this master.